float32 center_x
float32 center_y
float32 size_x
float32 size_y
string class_name
float32 confidence